include/EnvelopeGraph.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * EnvelopeGraph.h - Displays envelope graphs
+ *
+ * Copyright (c) 2004-2009 Tobias Doerffel <tobydox/at/users.sourceforge.net>
+ * Copyright (c) 2024-     Michael Gregorius
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#pragma once
+
```

Comment:
I think it's a good step towards using the compiler to speed up builds. Even though it's not a standard most compilers support it, and for the case where someone tries to use one without support it will still build because of the guards.

Reply:
Nowhere else do we use both include guards and `#pragma once` in a single header, and I've never heard of anyone doing that either. This just looks like a simple oversight.

Reply:
You're right, we don't use it anywhere else because we always use normal include guards. I think this would be a good first step towards using pragma once though.